scripts/variantstore/wdl/CreateCohortTable.wdl

```diff
@@ -0,0 +1,91 @@
+version 1.0
+
+workflow CreateCohortTable {
+   input {
+        String project
+        String dataset
+
+        String? docker
+    }
+
+    # TODO update this docker source
+    String docker_final = select_first([docker, "us.gcr.io/broad-dsde-methods/variantstore:latest"])
+
+    call CreateCohortTableTask {
+        input:
+            project                         = project,
+            dataset                         = dataset,
+
+            docker                          = docker_final
+    }
+
+}
+
+task CreateCohortTableTask {
+    # indicates that this task should NOT be call cached
+    meta {
+       volatile: true
+    }
+
+    input {
+        String project
+        String dataset
+
+        String? query_project
+        String? destination_project
+        String? destination_dataset
+
+        String? destination_cohort_table_name
+        String? fq_cohort_sample_table
+        String? fq_sample_mapping_table
+
+        File? service_account_json
+        String docker
+    }
+
+    #### set defaults ####
+    String query_project_final = if defined(query_project) then "${query_project}" else "${project}"
+    String destination_project_final = if defined(destination_project) then "${destination_project}" else "${project}"
+    String destination_dataset_final = if defined(destination_dataset) then "${destination_dataset}" else "${dataset}"
+
+    String destination_cohort_table_name_final = if defined(destination_cohort_table_name) then "${destination_cohort_table_name}" else "exported_cohort_all_samples"
+    String fq_cohort_sample_table_final = if defined(fq_cohort_sample_table) then "${fq_cohort_sample_table}" else "${project}.${dataset}.sample_info"
+    String fq_sample_mapping_table_final = if defined(fq_sample_mapping_table) then "${fq_sample_mapping_table}" else "${project}.${dataset}.sample_info"
+
+    String has_service_account_file = if (defined(service_account_json)) then 'true' else 'false'
+
+    command <<<
+        set -e
+
+        if [ ~{has_service_account_file} = 'true' ]; then
+          SA_FILENAME="sa_key.json"
+          gsutil cp "~{service_account_json}" $SA_FILENAME
```

Comment:
If this is declared as a`File` above, won't it already be copied down to the VM now and the path be a local path?